css/profilepage.css

```diff
@@ -120,9 +120,63 @@ a.button:active {
 }
 
 body {
-  background-image: linear-gradient(to right, blue, skyblue);
+  background-image: linear-gradient(to right, blue, rgb(70, 146, 176));
 }
 
 footer {
   background-color: aliceblue;
 }
+
+
+.profile-actions {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 10px;
+  
+}
+
+.profile-actions img {
+  width: 50px;
+  height: 50px;
+  
+}
+.rating-box{
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (code_refinement):** Duplicate CSS block for .rating-box.

The .rating-box class is defined twice with the same padding-top property. Consider merging them into a single block to clean up the CSS.